gcalbot/gcalbot/html.go

```diff
@@ -146,6 +136,93 @@ const tmplHeader = `<!DOCTYPE html>
 		height: 150px;
 		margin-bottom: 24px;
 	}
+
+	.select-container {
+		position: relative;
+		width: 256px;
+	}
+	.select-container select {
+		appearance: none;
+		-moz-appearance: none;
+		-webkit-appearance: none;
+
+		width: 100%;
+
+		background-color: white;
+
+		border-color: rgba(0, 0, 0, 0.1);
+		border-radius: 4px;
+		border-style: solid;
+		border-width: 1px;
+
+		padding-top: 8px;
+		padding-bottom: 8px;
+		padding-left: 16px;
+		padding-right: 32px;
+
+		font-size: 18px;
+		white-space: nowrap;
+		text-overflow: ellipsis;
+	}
+	.select-container:hover select {
+		// blue
+		border-color: #4C8EFF;
+	}
+	.select-container .caret {
+		display: flex;
+		position: absolute;
+
+		top: 0;
+		bottom: 0;
+		right: 0;
+
+		align-items: center;
+
+		padding-left: 8px;
+		padding-right: 16px;
+	}
+	.select-container:hover .caret {
+		// blueDark
+		fill: #3663EA;
+	}
+
+	//.checkbox-container {
+	//	height: 13px;
+    // 	width: 13px;
+	//
+	//	user-select: none;
+	//	-moz-user-select: none;
+	//	-webkit-user-select: none;
+	//	-o-user-select: none;
+	//	-ms-user-select: none;
+	//}
+	//
+	//.checkbox-container input {
+	//	appearance: none;
+	//	-moz-appearance: none;
+	//	-webkit-appearance: none;
+	//
+	//	height: 100%;
+	//	width: 100%;
+	//
+	//	background-color: rgb(255, 255, 255);
+	//	border-color: rgba(0, 0, 0, 0.2);
+	//	border-radius: 2px;
+	//	border-style: solid;
+	//	border-width: 1px;
+	//}
+	//
+	//.checkbox-container input:focus {
+	//	outline: 0;
+	//}
+	//
+	//.checkbox-container input:checked {
+	//	background-color: rgb(76, 142, 255);
+	//	border-color: rgb(76, 142, 255);
+	//	border-radius: 2px;
+	//	border-style: solid;
+	//	border-width: 1px;
+	//}
```

Comment:
maybe remove these styles if they're not being used?

Reply:
Will do. However, do you think that the button style should be changed? cc @joshblum

Reply:
woudl be nice if the button style was consistent across platforms esp mobile